repose-aggregator/src/docs/asciidoc/style-guide.adoc

```diff
@@ -55,4 +55,9 @@ Nesting in lists will correspond to the number of leading asterisks.
 ....
 
 == Citations
-* http://asciidoctor.org/docs/asciidoc-recommended-practices/[AsciiDoc Recommended Practices]
\ No newline at end of file
+* http://asciidoctor.org/docs/asciidoc-recommended-practices/[AsciiDoc Recommended Practices]
+
+== Repose is Bold
```

Comment:
Could/should this be generalized to all products? So *Docker* or *OpenShift* or other such things?

Reply:
I don't know. I took it more as we were promoting us and our direct use of other technologies.

That said, I would say no. However, I'm open for discussion.